src/core/annotation.js

```diff
@@ -3509,6 +3546,16 @@ class InkAnnotation extends MarkupAnnotation {
 
     results.push({ ref: inkRef, data: buffer.join("") });
   }
+
+  static async createAnnotationForPrinting(annotation, xref) {
```

Comment:
Similar to existing naming, how about:
```suggestion
  static async createNewPrintAnnotation(annotation, xref) {
```